.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: android
```

Comment:
To [provide the default infrastructure](https://docs.travis-ci.com/user/getting-started/#Selecting-infrastructure-(optional)) for builds to run on, add 
```yaml
sudo: false 
```